scraper/company_scraper_test.go

```diff
@@ -0,0 +1,213 @@
+package scraper
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestReadCompanyNames(t *testing.T) {
```

Comment:
Am having a challenge with testing the GetSearchResults function because of the API KEY, apparently that test keeps failing with the error: "SERPAPI_KEY not set in config or environment". I need some help here

Reply:
Can set this with `export SERPAPI_KEY=key_string` prior to running any tests that need this key